### PR TITLE
DESENG-55

### PR DESCRIPTION
**Pull request by Baelx**
Should be the last PR for this ticket in this repo. 

Completed file upload modal. Allow project to save logos. Clean up add-edit-project code

[-] Wrote a test or tests for code you added.
[x] Linted the files you touched.
[x] Wrote a doc block for each method you added.
[-] Add documentation to the README

### CODE REVIEW

**Review by SteveMHoward — COMMENTED** [review]
Changing method names (i.e. selectFile -> handleSelect) in older code makes me incredibly nervous. Are you confident every instance has been updated?

The risk here is that some other old piece of code isn't updated to use the new method, and we are left in a worst state then we started in.

File: src/app/projects/add-edit-project/add-edit-project.component.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Component, OnInit, AfterViewInit, ChangeDetectorRef, OnDestroy } from '@angular/core';
```
Comment (CR-gbc) [inline src/app/projects/add-edit-project/add-edit-project.component.ts:1]: this diff is so brutal, github tried to hide it from me 🤣

**Review by CR-gbc — APPROVED** [review]
the method name refactoring doesn't bother me, but do check the sourcecode for any references to those old calls.
this PR was large enough that I glossed over more boiler-platey-looking parts. you may want to await another review.

**Comment by Baelx** [comment]
> Changing method names (i.e. selectFile -> handleSelect) in older code makes me incredibly nervous. Are you confident every instance has been updated?
> 
> The risk here is that some other old piece of code isn't updated to use the new method, and we are left in a worst state then we started in.

Thanks for the review @SteveMHoward @CR-gbc .

Getting nervous about changing function names is totally valid. I did make sure that every instance of this call was changed. FYI the `selectFile` method was added a couple of commits ago so it's not an old method.

**Comment by SteveMHoward** [comment]
> 

Perfect :) I know how easy it is to miss one or two but looks like you fully tested it all.

**Pull request merged**